app-office/libreoffice/libreoffice-4.4.9999.ebuild

```diff
@@ -241,22 +246,6 @@ DEPEND="${COMMON_DEPEND}
 	test? ( dev-util/cppunit )
 "
 
-PATCHES=(
```

Comment:
Just to be clear, you could've left PATCHES and used `epatch "${PATCHES[@]}"` in `src_prepare()` but that's your + @akhuettel's decision ;-). Whichever is more convenient to you. Alternatively, the `PATCHES` array could also be a `local` in `src_prepare()`. So many possibilities.